lib/block_view.go

```diff
@@ -2759,6 +2801,42 @@ func (bav *UtxoView) _deleteRecloutEntryMappings(recloutEntry *RecloutEntry) {
 	bav._setRecloutEntryMappings(&tombstoneRecloutEntry)
 }
 
+func (bav *UtxoView) _addPostEntryPinning(postEntry *PostEntry) {
```

Comment:
can we make this `_setPinEntryMappings(publicKey []byte, postHash *BlockHash, pinEntry *PinEntry) {`